tests/route_testing_conf.py

```diff
@@ -0,0 +1,7 @@
+"""
+A very basic python configuration file.
+Our single source of truth for which
+routes need to be tested and their expected
+content.
+"""
+routes_and_test_content = {"/": b"Hello World", "/404": b"Page not found"}
```

Comment:
Very clean implementation - for future use (and discussion) perhaps this should be updated to enable specific tags to be checked for content potentially ?